scripts/publish.sh

```diff
@@ -18,21 +18,21 @@ packages=(
   # "playground" # NOT PUBLISHING
 )
 
-npm config set '//registry.npmjs.org/:_authToken' "${NPM_TOKEN}"
+# npm config set '//registry.npmjs.org/:_authToken' "${NPM_TOKEN}"
 
 pnpm build:packages
 
 for package in "${packages[@]}"
 do
   cd "packages/$package"
 
-  if [ "$CANARY" = "true" ]; then
-    pnpm publish --no-git-checks --tag canary
-  else
-    pnpm publish
-  fi
+  # if [ "$CANARY" = "true" ]; then
```

Comment:
Why comment this?

Reply:
Sorry about that. It was for debugging purposes. I have uncommented it.

Reply:
NP